index.html

```diff
@@ -86,5 +86,197 @@ <h1>I'm Stephen Glad to see you</h1>
+            <h2 class="project-title">Tonic</h2>
+            <div class="project-meta flex">
+              <p class="bold">Canopy</p>
+              <div class="round"></div>
+              <p>Backend</p>
+              <div class="round"></div>
+              <p>2015</p>
+            </div>
+            <div class="project-note">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="project-tag">
+              <ul class="project-skills flex">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JAVASCRIPT</li>
+              </ul>
+            </div>
+            <button>See Project</button>
+          </div>
+         </div>
+
+          <div class="project-card flex">
+            <img src="./images/Snapshoot Portfolio (2).svg" alt="project card image">
+            <div class="prject-card-info flex column">
+              <h2 class="project-title">Tonic</h2>
+              <div class="project-meta flex">
+                <p class="bold">Canopy</p>
+                <div class="round"></div>
+                <p>Backend</p>
+                <div class="round"></div>
+                <p>2015</p>
+              </div>
+              <div class="project-note">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+              <div class="project-tag">
+                <ul class="project-skills flex">
+                  <li>HTML</li>
+                  <li>CSS</li>
+                  <li>JAVASCRIPT</li>
+                </ul>
+              </div>
+              <button>See Project</button>
+            </div>
+          </div>
+
+          <div class="project-card flex">
+            <img src="./images/Snapshoot Portfolio (3).svg" alt="project card image">
+            <div class="prject-card-info flex column">
+              <h2 class="project-title">Tonic</h2>
+              <div class="project-meta flex">
+                <p class="bold">Canopy</p>
+                <div class="round"></div>
+                <p>Backend</p>
+                <div class="round"></div>
+                <p>2015</p>
+              </div>
+              <div class="project-note">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              </div>
+              <div class="project-tag">
+                <ul class="project-skills flex">
+                  <li>HTML</li>
+                  <li>CSS</li>
+                  <li>JAVASCRIPT</li>
+                </ul>
+              </div>
+              <button>See Project</button>
+            </div>
+          </div>
+      </div>
+    </section>
+    <section id="about">
+      <div class="about-container flex column">
+        <div class="abt-me flex column">
+          <h1 class="header-text">About Myself</h1>
```

Comment:
- [ ] Kindly remove the `border-radius` from the bottom of the works section and add it to the top right of the about section to match the design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205634480-f30dcb7a-cadc-43f7-a318-a9fb527c4b25.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205634548-9cb0855d-e705-47fe-a84a-201f40cf1fd0.png)